packages/realm-web/package.json

```diff
@@ -43,7 +43,8 @@
   "dependencies": {
     "@types/bson": "^4.0.0",
     "bson": "4.0.0",
-    "js-base64": "^2.6.3"
+    "js-base64": "^2.6.3",
+    "realm-network-transport": "^0.6.0"
```

Comment:
should this be updated to 0.7

Reply:
This is super strange - I thought that was what I did. And it should remain a development dependency, since it'll get bundled in. Thanks for catching this!

Reply:
I'm confusing myself here ... it needs to be v0.7.0 now that https://github.com/realm/realm-js/pull/3163 merged - I'll rebase and fix this.